robotd/usb.py

```diff
@@ -95,6 +98,8 @@ def control_read(self, request, value, index, length, timeout=3000):
 
 
 def enumerate():
```

Comment:
Not really related to your change, but could this function be renamed (`enumerate_devices`), I dont like overriding builtin functions like this

Reply:
Agreed, but would also agree not related to this PR :P

Reply:
Done!